theme/snippets/nav.liquid

```diff
@@ -29,7 +29,6 @@
       </a>
     </div>
     <a class="uppercase py1_875" href="/" aria-label="{{ 'snippets.nav.links.home_link.aria_label' | t }}">{{ 'snippets.nav.links.home_link.text' | t }}</a>
-    <a class="uppercase pb1_875" href="/" aria-label="{{ 'snippets.nav.links.knowledge_areas_link.aria_label' | t }}">{{ 'snippets.nav.links.knowledge_areas_link.text' | t }}</a>
     <a class="uppercase pb1_875" href="/pages/about" aria-label="{{ 'snippets.nav.links.about_link.aria_label' | t }}">{{ 'snippets.nav.links.about_link.text' | t }}</a>
```

Comment:
Removing this from the nav since we are no longer showing Knowledge Areas in the MVP